src/main/java/com/purbon/kafka/topology/backend/FileBackend.java

```diff
@@ -39,8 +36,8 @@ public void createOrOpen() {
   @Override
   public void createOrOpen(Mode mode) {
     try {
-      writer = new RandomAccessFile(filename(), "rw");
-      Path path = Paths.get(filename());
+      writer = new RandomAccessFile(STATE_FILE_NAME, "rw");
```

Comment:
actually, the idea of using a function was to allow changing the filename in case of need and to compose it from the configs with a different path. By adding a constant, and nothing else, this power is lost, right?

Reply:
Yes, but it wasn't used, as far as I could see. It may easily be introduced again later, if the need rises. My main goal was to get rid of the repeated ".cluster-state" from the tests.

Reply:
agree, it makes sense. Let's do it like this.